osx/unrar/license.txt

```diff
@@ -1,34 +1,127 @@
- ******    *****   ******   UnRAR - free utility for RAR archives

```

Comment:
I think we should keep the original license file, the new one feels inappropriate.